agent/xds/delta.go

```diff
@@ -180,6 +174,12 @@ func (s *Server) processDelta(stream ADSDeltaStream, reqCh <-chan *envoy_discove
 				}
 			}
 
+			if handler, ok := handlers[req.TypeUrl]; ok {
```

Comment:
Need to move this to _after_ the version-sniff so we can use the detected version to change behavior.